src/modules/bicycles/pages/RegisterBike/components/RegisterBikeForm/components/UploadImage.tsx

```diff
@@ -0,0 +1,86 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Mesmo caso do `RegisterBikeForm.tsx`

Reply:
Neste caso se faz necessário par forçar a label tag a ter um controlador asossiado.